src/ppspp_have.erl

```diff
@@ -48,7 +48,7 @@
 
 unpack(Addressing_Method, Message) ->
     {Chunk_Spec, Maybe_Messages} = ppspp_chunk:unpack(Addressing_Method, Message),
```

Comment:
Line 50 is too long:     {Chunk_Spec, Maybe_Messages} = ppspp_chunk:unpack(Addressing_Method, Message),.